app/views/dashboards/_table_content.html.erb

```diff
@@ -42,9 +42,20 @@
             <% end %>
           <% end %>
 
-          <td><%= link_to call_glyphicon, "#call-#{patient.primary_phone_display}", data: {toggle: "modal"} %></td>
+      <td>
+        <%= link_to call_glyphicon, "#call-#{patient.primary_phone_display}", class:"call_button",
+        data: {
+          patientname: patient.name,
```

Comment:
> That said, I worry that we're basically moving the 100 different modals into 100 different data fields and not actually generating any less code with this solution.

This was the big whoops, I realized this when all the ```data``` variables were called regardless. I wasn't sure how the data attribute worked at first.

> It looks like this is basically overruling what's in calls/_new_call with a bit of additional javascript to pop in the modal, and I do think there's some merit to keeping that as the flow -- slightly more restful.

Yup. I'm brute forcing JS in here ha.

My original idea was to generate a Rails method that calls all the helper methods, but controllers cannot call helper methods. So I went a little off rails..

Based on what you said and reading those guides, I am to generate a Rails method that calls ```js.erb```, render a partial which calls all the helper methods.

Can you show me a snippet where notes#create is being called?